docker/lib/dependabot/docker/file_parser.rb

```diff
@@ -124,15 +128,23 @@ def docker_repo_name(image, registry)
         "library/#{image}"
```

Comment:
@jakecoffman I needed your suggestion here, for the ecr container registry (AWS), if the replaces-base is set to true and Dockerfile contains the below, then the function `docker_repo_name` will return the repo name as  "library/docker-php". I guess adding "library" to the repo name is a docker hub thing. Should I need to add a check here to check if the registry passed with replaces-base == true is ecr or not and if replaces-base == true and the registry is ecr then simply return `docker-php` instead of appending `library` to it?
 
```
FROM docker-php@sha256:18305429afa14ea462f810146ba44d4363ae76e4c8dfc38288cf73aa07485005
```

Reply:
Yeah that makes sense, check the URL if the host contains `.ecr.` I guess? That makes sense I think!

Reply:
I thought more on this and I think this is also applicable for all private registries passed with replaces-bases. Only when registry is default `https://registry.hub.docker.com` then we should append `library/` to the repo?